do-app-platform-op-cli/README.md

```diff
@@ -1,314 +1,285 @@
   ```
 
-  Using Powershell:
+**Powershell**:
+
   ```pwsh
   Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml | op inject | doctl apps create --spec - --wait --upsert
   ```
 
-  </details>
-
-## Appendix
-
-### Supply custom vault and item names
-
-If you chose your own name for the vault and items where you saved your SCIM bridge credentials, you can override the defaults using the `VAULT` and `ITEM` variables in the secret references. For example:
-
-Using Bash:
-```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
-```
+</details>
 
-Using Powershell:
-```pwsh
-Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
-```
+## Update your SCIM bridge
 
-### Update 1Password SCIM bridge
+To update your SCIM bridge, run the following command for your shell.
 
-The latest version of 1Password SCIM bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. The most recent version should also be pinned in [`op-scim-bridge.yaml`](./op-scim-bridge.yaml), so you can update using the same command as above with the `--upsert` parameter:
+**Bash**:
 
-Using Bash:
 ```sh
 curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
 
-Using Powershell:
+**Powershell**:
+
 ```pwsh
 Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
 
-### Propose the app spec
-
-You can optionally propose the raw app spec template to verify the cost before deploying to DigitalOcean:
-
-Using Bash:
-```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
-```
-
-Using Powershell: 
-```pwsh
-Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
-```
+The [op-scim-bridge.yaml](./op-scim-bridge.yaml) file that this command uses should have the latest version in it. If the version doesn't match the one on the [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM), you can download the file and update it before you run the command.
```

Comment:
Do we want to start pointing folks here now that we also put SCIM bridge notes on the newer site?

```suggestion
The [op-scim-bridge.yaml](./op-scim-bridge.yaml) file that this command uses should have the latest version in it. If the version doesn't match the one on the [SCIM Bridge Release Notes page](https://releases.1password.com/provisioning/scim-bridge/), you can download the file and update it before you run the command.
```

Reply:
That's a great point! I can add this to the other files as well.